### DISCUSSION TITLE
Using React Table with IDOM

### DISCUSSION THREAD

**Discussion by Archmonger** [idom-team/idom#540] [Question] [answered]
Does IDOM natively support exporting react hooks to Python?

For example, the [usePagination](https://react-table.tanstack.com/docs/api/usePagination) hook within React-Table.

**Comment by rmorshea** [comment, marked as answer]
It does not unfortunately.

**Reply by Archmonger** [reply]
Is it feasible to automate the conversion of (some/most) third party hooks into IDOM-compatible hooks via AST parsing+transformations?

**Reply by rmorshea** [reply]
It looks like there's a package called Js2Py that could help with that, but it'd still require some amount of manual intervention I imagine.

**Reply by Archmonger** [reply]
I'll draft a change request so we can look at the feasibility of this. I'd say investigating should belong on the v2 tasks.
